### PR TITLE
View more cleanup and a11y fixes

### PR DESCRIPTION
**Pull request by frrrances**
This PR contains changes styles for the view more page (along with some adjustments to generic layout class names), makes the item buttons more prominent, and adds some small text changes to make the UI consistent.

And gem updates.

### CODE REVIEW

File: config/environments/development.rb [hunk 1]
```diff
@@ -51,4 +51,7 @@
   # Use an evented file watcher to asynchronously detect changes in source code,
   # routes, locales, etc. This feature depends on the listen gem.
   # config.file_watcher = ActiveSupport::EventedFileUpdateChecker
+
+  # Whitelist web console
+  config.web_console.whitelisted_ips = '10.0.0.0/8'
```
Comment (JPrevost) [inline config/environments/development.rb]: This is fine for now, but we may want to consider options to set this outside of the repo with an ENV flag or something.

File: app/assets/stylesheets/bento.scss [hunk 2]
```diff
@@ -376,3 +378,28 @@ $z-depth-way-front:     1000;
...
+
+.subline {
+  font-size: $fs-xsmall;
+}
+
+.nav .current {
+  font-weight: $fw-bold;
+}
+
+// changes for testing - TODO: remove
```
Comment (JPrevost) [inline app/assets/stylesheets/bento.scss]: I'll assume you have a good reason for commit commented out code... if you don't 👎 :)

Comment (frrrances) [inline app/assets/stylesheets/bento.scss]: i need that to hide the source when testing - i think it's helpful for anyone developing to have that handy so I put a TODO on it and am adding it to the repo. I'll remove it when we don't need source any more for development. Cool?

Comment (JPrevost) [inline app/assets/stylesheets/bento.scss]: I sorta want to hide this by default and have a debug mode that turns it back on. But we can do that later.

File: app/views/search/_result.html.erb [hunk 3]
```diff
@@ -74,7 +74,7 @@
   
   <% if result.url %>
     <div class="result-get">
-      <%= link_to('Get it', result.get_it_url, class: 'button button-secondary') %>
+      <%= link_to('Get it', result.get_it_url, class: 'button button-primary') %>
```
Comment (JPrevost) [inline app/views/search/_result.html.erb]: FYI: we're already setting the `result.get_it_url` in the normalizer classes. We can easily create and set a `result.get_it_label` to give us more control over the labeling.

File: app/views/search/search_boxed.html.erb [hunk 4]
```diff
@@ -1,59 +1,53 @@
...
+        <%= paginate(Kaminari.paginate_array(@results['results'], total_count: max).page(params[:page]).per(10)) %>
+      </div>
+    <% end %>
 
 <% if current_page?(:controller => 'search', :action => 'search') %>
+    </div>
+    <div class="col1q-r">
+    <div class="bit">
+      <h3>Supporting content</h3>
+      <p>Lorem ipsum dolor sit amet, consetetur sadipscing elitr: </p>
```
Comment (JPrevost) [inline app/views/search/search_boxed.html.erb]: Is it okay if I feel this content is better than a lot of our published stuff?

File: app/views/search/_result.html.erb (hunk 3 above)
Comment (frrrances) [inline app/views/search/_result.html.erb]: excellent!

File: app/views/search/search_boxed.html.erb (hunk 4 above)
Comment (frrrances) [inline app/views/search/search_boxed.html.erb]: 🌈

**Pull request merged**